scripts/checkYaml.lean

```diff
@@ -4,7 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Eric Wieser
 -/
 
-import Mathlib.Tactic.Basic
+import Std.Tactic.Lint
```

Comment:
This one seems strange to me; is it really minimal, despite this file not being about linting?

Reply:
It has a `open Std.Tactic.Lint` in it, I didn't investigate why. This import is not precisely minimal, `Std.Tactic.Lint` is the root of a small number of other files

Reply:
Apparently the `open Std.Tactic.Lint` was unnecessary and after removing it the `import Std.Tactic.Lint` could be removed as well.